javascript-data-model/order-history.js

```diff
@@ -0,0 +1,366 @@
+        noteToCustomer: 'Your package was delivered. It was handed directly to a resident.'
+      }
+    }
+  ],
+  [
+    {
+      productDetails: {
+        title: 'Gamecube Controller Adapter. Super Smash Bros Switch Gamecube Adapter for WII U, PC. Support Turbo and Vibration Features. No Driver and No Lag-Gamecube Adapter',
+        author: {
+          title: '',
+          firstName: '',
+          lastName: ''
+        },
+        image: 'random image',
+        price_b4_SH: 15.98,
+        returnWindow: {
+          day: 5,
+          month: 8,
+          year: 2020
+        },
+        productReview: {
+          stars: 5,
+          featureRating: {
+            feature1: 5,
+            feature2: 2,
+            feature3: 4
+          },
+          photo: 'photo',
+          video: '',
+          review: {
+            headline: 'OMG I LAAAV IT',
+            reviewWriting: 'Buy this, you need it!'
+          }
+        }
+      },
+      orderDetails: {
+        orderStatus: {
+          ordered: {
+            status: true,
+            date: {
+              day: 4,
+              month: 7,
+              year: 2020
+            }
+          },
+          shipped: {
+            status: true,
+            date: {
+              day: 5,
+              month: 7,
+              year: 2020
+            }
+          },
+          outForDelivery: {
+            status: true,
+            date: {
+              day: 6,
+              month: 7,
+              year: 2020
+            }
+          },
+          estimatedArival: {
+            status: true,
+            date: {
+              day: 7,
+              month: 7,
+              year: 2020
+            }
+          },
+          delivered: {
+            status: true,
+            date: {
+              day: 7,
+              month: 7,
+              year: 2020
+            }
+          }
+        },
+        orderNum: '114-2875557-9059409',
+        shipTo: 'JS Masher',
+        invoice: {},
+        orderPriceTotal: 17.22,
+        noteToCustomer: 'Your package was delivered. It was handed directly to a resident.'
+      }
+    }
+  ],
+  [
+    {
+      product1: {
+        productDetails: {
+          title: 'GameCube Controller - Super Smash Bros. Edition (Nintendo Switch)',
+          author: {
+            title: '',
+            firstName: '',
+            lastName: ''
+          },
+          image: 'random image',
+          price_b4_SH: 94.95,
+          returnWindow: {
+            day: 4,
+            month: 8,
+            year: 2020
+          },
+          productReview: {
+            stars: 5,
+            featureRating: {
+              feature1: 5,
+              feature2: 2,
+              feature3: 4
+            },
+            photo: 'photo',
+            video: '',
+            review: {
+              headline: 'OMG I LAAAV IT',
+              reviewWriting: 'Buy this, you need it!'
+            }
+          }
+        }
+      },
+      product2: {
```

Comment:
Instead of doing individually named properties for each item, I recommend using an array that holds all items for the order. Make sure you use the same format in your other orders as well!